content/S2016/S16-event-csters-destress.md

```diff
@@ -0,0 +1,22 @@
+title: Big CSters: WiCS & FemPhys De-stressing Session
+Date: 2016-6-1
+Category: Events
+Tags: social, big csters
+Slug: big-csters-destressing-s16
+Author: Clare Ng
+Summary: Come join us for a relaxing night of socializing and de-stressing.
+
+WiCS is collaborating with FemPhys to bring you a de-stressing session! We 
+know school can be pretty stressful sometimes, so hopefully you can take 
+an opportunity to take a breather and de-stress with us for a couple hours.
+
+As always, food will be provided! (With vegetarian, vegan, gluten-free options.)
+
+Note: This event is [women-only](http://wics.uwaterloo.ca/faq).
+
+## Event Details ##
+
++ **Who:** [Women and non-binary individuals](http://wics.uwaterloo.ca/faq).
++ **What:** WiCS & FemPhys De-stressing Session
++ **Where:** TBA
++ **When:** Tuesday. June 7, 6:00 PM
```

Comment:
we now know it'll go until 7:30

@techiechelle what kind of details can we add? Face masks, discussion, anything else?